source/pool/contracts/Pool.sol

```diff
@@ -128,34 +131,55 @@ contract Pool is IPool, Ownable2Step {
   /**
    * @notice Set claims from previous pool contract
    * @dev Only owner
-   * @param _root bytes32
+   * @param _proposalId bytes32
    * @param _accounts address[]
    */
   function setClaimed(
-    bytes32 _root,
+    bytes32 _proposalId,
     address[] memory _accounts
   ) external override multiAdmin {
-    if (roots[_root] == false) {
-      roots[_root] = true;
+    if (roots[rootIds[_proposalId]] == false) {
+      roots[rootIds[_proposalId]] = true;
     }
     for (uint256 i = 0; i < _accounts.length; i++) {
       address account = _accounts[i];
-      if (claimed[_root][account]) revert AlreadyClaimed();
-      claimed[_root][account] = true;
+      if (claimed[rootIds[_proposalId]][account]) revert AlreadyClaimed();
```

Comment:
This could cause an issue - we switched to `proposalId` instead of root to avoid the case where two proposals share the same `root` (the same voters, each with the same points - this will be more common than it sounds because sometimes there are multiple proposals for the same snapshotted block).

Reply:
Done in the new commit